python/tvm/expr.py

```diff
@@ -18,32 +18,54 @@
 from __future__ import absolute_import as _abs
 from ._ffi.node import NodeBase, NodeGeneric, register_node
 from . import make as _make
+from . import generic as _generic
 from . import _api_internal
 
 class ExprOp(object):
     def __add__(self, other):
-        return _make.Add(self, other)
+        try:
+            return _generic.add(self, other)
```

Comment:
Ideally; we should be able to use generic.add directly for Exprs(put implementation in generic)